### ISSUE TITLE
Adding more commands in the Script > Insert dialog Commands Tab

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#8696]
@Patowhiz I am really liking this simple addition.  I hope you do too.  It is nowhere near as important as the addition of filters, loops etc, but useful and should be easy for anyone to do.
@N-thony assuming you and Patrick agree can you allocate?  It would be very quick for you to do, and also (I hope) also easy for others.

A really simple one to add is the file.choose() command that is in base R.  I suggest it is an excellent addition for us in the Insert > Commands tab.

![image](https://github.com/africanmathsinitiative/R-Instat/assets/11226469/93d18094-e42c-4cb5-a89a-494a936d6036)

It goes just below the Get Package option.

The option label is `Choose file `  
It is followed by a checkbox (default unchecked) with label `Open R file`

If unchecked then it produces the comment and code as follows:
```
#Open file interactively
filename <- file.choose()

```
If checked it produces:
```
#Open R data file interactively
filename <- file.choose()
datafile <- readRDS(filename)
```
Note I first found this suggestion [here](https://stackoverflow.com/questions/59107717/how-do-i-load-an-rds-file-into-r).  
What's interesting is that there is a comment afterwards that this is a terrible thing to do, because it produces a script that is to some extent interactive.  And you really, really don't want that in scripts you produce for others.  

This complaint ties well with [this video](https://www.google.com/search?q=where+is+file.choose+in+R&oq=where+is+file.choose+in+R&gs_lcrp=EgZjaHJvbWUyBggAEEUYOTIHCAEQIRigATIHCAIQIRigATIHCAMQIRigATIHCAQQIRigATIKCAUQIRgWGB0YHjIKCAYQIRgWGB0YHjIKCAcQIRgWGB0YHjIKCAgQIRgWGB0YHtIBCjEyMTcyajBqMTWoAgCwAgA&sourceid=chrome&ie=UTF-8) on using file.choose in RStudio.   

In the video, the file.choose is used in the console, so the fixed resulting filename can be in the script.  It is a good example of why a script in R-Instat is different to RStudio!

**Comment by rdstern** [comment]
@N-thony good point.  But I was assuming that you might occasionally want to open an R file at the start of a script, because it has been sent to you.  If you want to open any file, then use our File > Import dialog.  If you then want to open that file from a script, then use the To script botton to add it to your code.

@derekagorhom , I hope you two can add this task to your list?

**Comment by Patowhiz** [comment]
@rdstern 

I have reservations about the idea of incorporating additional standard R commands into the dialog. This seems to deviate from the original purpose of the dialog unless we collectively agree to redefine its goal.

My understanding of the dialog's objective is to assist users in learning about data book custom functions. By providing users with a set of controls that generate data book functions used by other dialogs behind the scenes, we aim to make them familiar with these functions and how the data book `object` maintains its "data state."

Anything beyond the scope of the data book might be better directed elsewhere (e.g., R documentations, Google, etc.) to avoid cluttering the dialog and making it overly complex for users.

For example, the functionality you've suggested would likely be most useful for users already familiar with standard R. For them, it might be simpler to type or paste the commands instead of navigating to the dialog to have it generate them. The same rationale applies to the Get Package control.

That being said, I'm open to your objections or suggestions if you believe strongly in adding this feature.
